eclipse-collections-code-generator/src/main/resources/impl/stack/mutable/primitiveArrayStack.stg

```diff
@@ -151,12 +151,7 @@ public class <name>ArrayStack extends Abstract<name>Stack
         this.delegate.clear();
     }
 
-    @Override
```

Comment:
This looks like an unrelated change. Please revert and create an issue if this is duplication.

Reply:
It is related. Please see the PR comment and the issue comment. BooleanIterable does not have toSortedList() so to have a single template for primitive stacks it needs to be excluded from code generation for boolean (just like sortThis() is already excluded from mutable primitive list generation for booleans).

Reply:
Agree with you Vlad, however, such “baked in” changes make it difficult to write release notes. This is what I propose, break it up as an additional commit as a part of this same PR so that it reads as another commit.

Reply:
I see, thanks for clarifying.

Reply:
Wouldn't describe it as a “baked in” change. The actual change is to have BooleanArrayStack generated and this is necessary to make it happen. The template change preserves APIs for the other primitive stacks and for the boolean array stack, so there is really no change in behavior. BooleanArrayStack never supported toSortedList, toSortedList is a part of the common template, so we exclude toSortedList generation for booleans in the template